include/uv.h

```diff
@@ -1668,11 +1668,13 @@ union uv_any_req {
 };
 #undef XX
 
+typedef void(*uv_loop_cb)(uv_loop_t*);
 
 struct uv_loop_s {
   /* User data - use this for whatever. */
   void* data;
   /* Loop reference counting. */
+  uv_loop_cb on_watcher_queue_updated;
```

Comment:
Tiniest of nits: `watcher_queue_updated_cb` would be a more idiomatic name and I suggest moving it a few lines down to indicate it's internal.

If you shrink `reserved` by one and put it right before or after that data member, you could turn this into something that's semver-minor rather than semver-major.